src/smokeTest/java/uk/gov/hmcts/reform/demo/RootSmokeTest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.demo;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.restassured.RestAssured;
+import net.serenitybdd.rest.SerenityRest;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+public class RootSmokeTest {
+
+    private final String targetInstance =
+        StringUtils.defaultIfBlank(
+            System.getenv("TEST_URL"),
+            "http://localhost:8098"
```

Comment:
might want to change the port as it will clash with `ia-case-api` otherwise

Reply:
I thought ia-case-api is 8090. could you suggest please

Reply:
Ah yes it is.  Let's do `8094`

Reply:
Will do the change in separate PR.